readme.md

```diff
@@ -12,3 +12,4 @@ Individual Project Videos - Work in Progress
 ## Test Instruction
 
 1. npm run test or npx playwright test
+2. Run the webserver "node index.js"
```

Comment:
Hey, lets also update the package.json file so that it can start the web server with the command "npm run start"